build.gradle

```diff
@@ -62,9 +60,11 @@ buildscript {
 }
 
 plugins {
-  id 'ru.vyarus.animalsniffer' version '1.5.0'
-  id 'com.github.johnrengelman.shadow' version '5.2.0'
-  id 'me.champeau.gradle.japicmp' version '0.2.9'
+  id "ru.vyarus.animalsniffer" version "1.5.0"
```

Comment:
The ' quotes don't do interpolation, the " quotes do. Motivation here?

Reply:
I think that isn't how my brain works, I wouldn't use different strings based on whether it needs interpolation.  I'd just mainly use the interpolated form whether or not is has a variable part.